src/gateways/findWardByCode.js

```diff
@@ -0,0 +1,17 @@
+const recordToDomain = ({ id, code, trust_id }) => ({
```

Comment:
Do you want to put this into its own pg package to save refactoring later when we want to differentiate between postgres and mssql? Or will it be done as a large PR with the rest of the already separated out gateways?

Reply:
We can leave this how it is for now and move it into a package if we feel it's useful later.